Lecture-14/src/main/java/service/RestAssuredRepositoryService.java

```diff
@@ -0,0 +1,41 @@
+package service;
+
+import client.IRestClient;
+import constant.HttpMethodType;
+import deserializer.Deserializer;
+import model.GitHubResponse;
+import model.GitUser;
+
+import java.util.List;
+
+public class RestAssuredRepositoryService implements IRepositoryService {
```

Comment:
You should only have 1 `IRepositoryService ` implementation, that uses `IRestClient` instance.
Another implementation may be needed e.g. to take info directly from DB or read from file

Reply:
done